### ISSUE TITLE
Bug in porepy.fracs.structured.cart_grid_2d

### ISSUE THREAD

**Issue by pnsaevik** [pmgbergen/porepy#26]
The `cart_grid_2d` function fails if there is more than 1 intersection. Minimal example:

```
    frac1 = np.array([[1, 4], [2, 2]])
    frac2 = np.array([[2, 2], [1, 4]])
    frac3 = np.array([[3, 3], [1, 4]])
    fracs = [frac1, frac2, frac3]
    gb = porepy.fracs.structured.cart_grid_2d(fracs, [5, 5])
```

Solution: Line 229,

```
for global_node in np.where(shared_nodes > 1):
```

Should be changed to

```
for global_node in np.where(shared_nodes > 1)[0]:
```

(I could have fixed it myself if I had access...)

**Comment by keileg** [comment]
I'll take it from here.

**Comment by rbe051** [comment]
Fixed by latest commit (29b6db31547c7440f0c3801d8aed074d8a86ac37)
